index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trial Project</title>
+    <link rel="stylesheet" href="style.css">  
+</head>
+<body>
+
+    <header class="header">
+        <a href="#">
+        <img src="./images/youtube.png" alt="YouTube Logo" class="youtube-logo">
+        </a>
+        <form class="search-bar">
+            <input class="search-input" type="search" placeholder="Search" aria-label="Search" />
+            <button type="submit" class="search-btn">
+                <img class="search" src="./images/search.png" />
+            </button>
+        </form>
+        <div class="menu-icons">
+            <img class="mic" src="./images/mic.png" alt="Upload Video">
+        </div>
+    </header>
+    
+    <section>
+        <iframe class="iframe" width="760" height="490" src="https://www.youtube.com/embed/gXLjWRteuWI" 
+        title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; 
+        encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+        <div class="video-details">
+            <a href="#" class="video-title">Learn HTML and CSS Crash Course</a>
+        </div>
+        <div class="discription">
+            <span>12K views</span>
+            <h3 class="view">&#x2022;</h3>
+            <span class="span">3 weeks ago</span>
+        </div>
+    </section>
+
+    <section>
+        <div class="categories">
+            <section class="category-section">
+                <button class="category active">All</button>
+                <button class="category">Category 1</button>
+                <button class="category">Category 2</button>
+                <button class="category">Category 3</button>
+                <button class="category">Category 4</button>
+                <button class="category">Category 5</button>
+                <button class="category">Category 6</button>
```

Comment:
- kindly use Html semantic tags  ul and li instead of button.

- This is important for semantic and accessibility reasons